app/k9mail/src/main/AndroidManifest.xml

```diff
@@ -59,7 +60,8 @@
         android:label="@string/app_name"
         android:theme="@style/Theme.K9.Startup"
         android:resizeableActivity="true"
-        android:allowBackup="false">
+        android:allowBackup="false"
+        tools:targetApi="n">
```

Comment:
`targetApi` feels wrong here. Feel free to ignore the warning using `tools:ignore="UnusedAttribute"`.